### DISCUSSION TITLE
0.8.0-1

### DISCUSSION THREAD

**Discussion by LinusCDE** [LinusCDE/chessmarkable#17] [Releases]
This release adds the PGN Viewer contributed by @rmadhwal in PR #16 !

You can now add files and play them back in chessmarkable with this. More in the readme!

<hr /><em>This discussion was created from the release <a href='https://github.com/LinusCDE/chessmarkable/releases/tag/0.8.0-1'>0.8.0-1</a>.</em>

**Comment by maoherve** [comment]
Hi, do you know if chessmarkable is runable on RMPP ?
Thanks

**Reply by LinusCDE** [reply]
I have not tested it yet. All the graphics should be gray-scale, so you won't see any color with it (so not a bug). I'm currently not involved in remarkable development and have not looked into what did change hardware-wise.

It might be lucky and pretty similar to the rM 2. But the application will still likely panic, as it expects to find either rM 1 or 2 as the device model and won't find those.

But I'd assume, that once other rM-Hacks (especially gui apps) can run, that updating libremarkable (hw impl) and this app should be pretty much a no-brainer.
